.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
```

Comment:
I don't fully understand why, but this needs to be moved to make it run correctly:
https://github.com/nedbat/coveragepy/issues/512

Reply:
Personally, I prefer separate config files for different tools. This way, I know where to go on the file system and within a config, instead of trying out many possible locations.

Also, https://github.com/nedbat/coveragepy/issues/512#issuecomment-1246595148 mentions that it's best to specify `--cov-config=.coveragerc` explicitly.

Reply:
I think it'd be reasonable to have adjusting the coverage config in a separate PR.

Reply:
Here's a more comprehensive config, by the way: https://github.com/ansible/pylibssh/blob/devel/.coveragerc.